CorEstafette/Hubs/SignalRHub.cs

```diff
@@ -27,5 +39,18 @@ public async Task<IResponse> UnsubscribeTopicAsync(Message message)
             message.Content = $"{message.Sender} successfully unsubscribe from topic {message.Topic}";
             return new Response(message, true);
         }
+
+        public override Task OnDisconnectedAsync(System.Exception exception)
+        {
+            string userName = "";
+            foreach( var pair in UserConnections)
```

Comment:
Instead of finding the value with a loop, we could use Linq and do:
**UserConnections.TryRemove(UserConnections.Keys.First(key => UserConnections[key] == Context.ConnectionId), out _);**